syft/frameworks/crypten/context.py

```diff
@@ -10,7 +10,7 @@
 from crypten.communicator import DistributedCommunicator
 
 
-def _launch(func, rank, world_size, master_addr, master_port, queue, func_args, func_kwargs):
+def _launch(func, rank, world_size, master_addr, master_port, queue, func_args, func_kwargs): # pragma: no cover
```

Comment:
Needed because this is run on the remote worker and it seems is not seen by the coverage package :\